apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -54,6 +58,8 @@
   private static final Logger log = LoggerFactory.getLogger(ApiarySnsListener.class);
 
   private static final String TOPIC_ARN = System.getenv("SNS_ARN");
+  private static final String HKAAS_REGEX = System.getenv("HKAAS_REGEX");
```

Comment:
We should try avoid any reference to HKAAS in this project so they're not coupled. We happen to want to filter the table parameters for HKAAS but other people might want to filter other table parameters for a completely different use case so let's make this more generic. Why don't we call this "TABLE_PARAM_FILTER" and we then clearly document that you pass in a regular expression and any Hive table params that match the filter will go into the SNS message. If this value isn't set (the default) then no table params are sent at all.

Reply:
Changed to `TABLE_PARAM_FILTER`